nva-datamodel-java/src/test/java/no/unit/nva/model/file/FileModelTest.java

```diff
@@ -289,7 +290,7 @@ private File getPublishedFileWithCustomRRS(UUID identifier) {
             .withName(FIRST_FILE_TXT)
             .withPublisherAuthority(false)
             .withSize(SIZE)
-            .withRightsRetentionStrategy(CustomerRightsRetentionStrategy.create())
+            .withRightsRetentionStrategy(CustomerRightsRetentionStrategy.create(UNKNOWN))
```

Comment:
Or potentially Random element…

Reply:
There will be some rules enforced as to what are valid combinations.Not sure if I want to generate random elements that does not make sense here. Clients not setting the field should end up with DEFAULT until the object has been through the business logic and finally stored with a "valid" value in the database.